src/database/schema/UserSchema.ts

```diff
@@ -305,11 +305,52 @@ class UserSchema implements BaseDbSchema {
         id: uuidv4(),
         channelId,
         userId: json?.user?.id,
-        username: json?.user?.name,
+        username: json?.user?.username,
         countryCode: '',
         createdAt: json?.user?.created_at,
         updatedAt: json?.updated_at,
         lastActiveAt: json?.user?.last_active,
+        profilePicture: json?.user?.image || DEFAULT_PROFILE_PIC_PATH,
+        bio: '',
+        isBanned: json?.banned,
+        anon_user_info_color_code: json?.anon_user_info_color_code,
+        anon_user_info_color_name: json?.anon_user_info_color_name,
+        anon_user_info_emoji_code: json?.anon_user_info_emoji_code,
+        anon_user_info_emoji_name: json?.anon_user_info_emoji_name
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The error handling in the `fromMemberWebsocketObject` method is a good addition. However, it's important to note that logging the error to the console may not be sufficient in a production environment. Consider using a more robust logging solution or an error tracking service.

```diff
-      console.log('error on fromMemberWebsocketObject', e);
+      // Replace with your preferred logging or error tracking solution
+      logger.error('Error on fromMemberWebsocketObject', e);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->